icu4c/source/i18n/unicode/unum.h

```diff
@@ -401,6 +401,13 @@ typedef enum UNumberFormatFields {
     UNUM_MEASURE_UNIT_FIELD,
     /** @stable ICU 64 */
     UNUM_COMPACT_FIELD,
+#ifndef U_HIDE_DRAFT_API
+    /**
+     * Approximately sign. In ICU 70, this was categorized under the generic SIGN field.
+     * @draft ICU 71
+     */
+    UNUM_APPROXIMATELY_SIGN_FIELD,
+#endif // U_HIDE_DRAFT_API
```

Comment:
Do we need to change the formula of "UNUM_FIELD_COUNT" below afte we add this value?

Reply:
The policy is not completely clear to me, but it was an oversight that it got no update at all. I have now updated it conditional on `U_HIDE_DRAFT_API`.